nft_ingester/src/api/dapi/asset.rs

```diff
@@ -294,7 +296,7 @@ pub async fn get_by_ids<
                             },
                         );
                     }
-                    Err(e) => {}
+                    Err(_) => {}
```

Comment:
The parameter is unused, indeed. However, are completely swallowing the error looks dubious. Maybe should it be logged at least?

Reply:
or return Result<FullAsset, AppError> it should simplify code base and remove boilerplate code